src/tasks/level10.ts

```diff
@@ -131,46 +131,8 @@ export const GiantQuest: Quest = {
           if (have($skill`Emotionally Chipped`) && get("_feelEnvyUsed") < 3 && have($item`amulet of extreme plot significance`))
             return Macro.skill($skill`Feel Envy`).step(killMacro());
           return new Macro();
-        }, $monster`Burly Sidekick`)
-        .forceItems($monster`Quiet Healer`),
-    },
-    {
-      name: "Airship",
-      after: ["Airship YR Healer"],
-      completed: () => have($item`S.O.C.K.`),
-      do: $location`The Penultimate Fantasy Airship`,
-      choices: () => {
-        return { 178: 2 };
-      },
-      post: () => {
-        if (have($effect`Temporary Amnesia`)) cliExecute("uneffect Temporary Amnesia");
-      },
-      orbtargets: () => [],
-      outfit: () => {
-        if (get("_saberForceUses") < 5 && !have($item`Mohawk wig`) && !have($item`bat wings`)) {
-          return {
-            modifier: "-combat",
-            weapon: $item`Fourth of May Cosplay Saber`,
-            avoid: $items`Kramco Sausage-o-Matic™`,
-          };
-        } else {
-          return {
-            modifier: "-combat",
-            equip: $items`bat wings`,
-          };
-        }
-      },
-      limit: { soft: 50 },
-      delay: () =>
-        have($item`Plastic Wrap Immateria`) ? 25 : have($item`Gauze Immateria`) ? 20 : 15, // After that, just look for noncombats
-      combat: new CombatStrategy().macro(() => {
-        if (have($item`Mohawk wig`)) return new Macro();
-        if (have($skill`Emotionally Chipped`) && get("_feelEnvyUsed") < 3)
-          return Macro.skill($skill`Feel Envy`).step(killMacro());
-        if (haveEquipped($item`Fourth of May Cosplay Saber`) && get("_saberForceUses") < 5 && !have($item`bat wings`))
-          return Macro.skill($skill`Use the Force`);
-        return new Macro();
-      }, $monster`Burly Sidekick`),
+        }, $monsters`Quiet Healer, Burly Sidekick`)
+        .forceItems($monsters`Quiet Healer, Burly Sidekick`),
```

Comment:
ForceItems isn't ideal here because it will keep forcing even after the goal item has dropped, potentially wasting envies. This is really a weakness in grimoire CombatStrategy. That was why there was two tasks here, and why Burly Sidekick didn't go through forceItems.

Reply:
So, we have a problem.

If we do HITS skip we don't have enough Envies to do both sidekick and healer. If we use saber, the script will break. We can only force one.

Would you prefer a rule that forces whichever comes first then just finishes airship?

Reply:
If we can only force one then we should just force healer. I would say modify HiTS too but this PR is already too long. We can worry about that later.